Documentation/ProjectFile/prj/processes/process/THERMO_MECHANICS/t_solid_density.md

```diff
@@ -0,0 +1 @@
+Reference solid density. It changes with temperature difference.
```

Comment:
I'm not sure if I understand this correctly: the reference density is temperature dependent, is it?
Then please add the law describing the change.
Also I'm not sure if the @nagelt 's comment below was answered.